TestServer/build.gradle.kts

```diff
@@ -14,6 +16,24 @@ dependencies {
 }
 
 
+val register = tasks.register("startServer", CustomTask::class) {
+    dependsOn("jar")
+    classpath = files("build/libs/TestServer-1.0.0.jar")
+}
+
+
+tasks.register("stopServer"){
+    println("I should stop server but i'm int implemented what a disaster, isn't it ?")
```

Comment:
i'm int
what is it?

Reply:
fixed in https://github.com/Mazel-Tovr/kotlin-app/pull/6